products/memgraph/v0.15.0/quick-start.md

```diff
@@ -1,27 +1,24 @@
 ## Quick Start
 
 This article briefly outlines the basic steps necessary to install and run
-Memgraph. It also gives a brief glimpse into the world of OpenCypher and
-outlines some information on programmatic querying of Memgraph. The users
-should also make sure to read and fully understand the implications of
-[telemetry](#telemetry) at the very end of the article.
+Memgraph. It also introduces how to make queries using [OpenCypher](https://www.opencypher.org). Finally, please
+be aware of and understand the implications of
+[telemetry](#telemetry) as used by the Memgraph team.
```

Comment:
@gitbuda 
I've just now realized these links don't work on github, but they work on the web. I'll add a task on asana to try to find the solution for both. Even worse, the curly braces after headings are also rendered, I don't know how me missed that.